packages/ui/src/lib/alert/ErrorMessage.svelte

```diff
@@ -6,13 +6,26 @@ import Tooltip from '../tooltip/Tooltip.svelte';
 
 export let error: string;
 export let icon = false;
+export let actionErrorInfo = false;
 </script>
 
 {#if icon}
   {#if error !== undefined && error !== ''}
-    <Tooltip top tip={error}>
-      <Fa size="1.1x" class="cursor-pointer text-[var(--pd-state-error)] {$$props.class}" icon={faExclamationCircle} />
-    </Tooltip>
+    {#if actionErrorInfo}
+      <Tooltip left tip={error} tipWidth="w-max max-w-[650px] overflow-hidden text-wrap">
+        <Fa
+          size="1.1x"
+          class="cursor-pointer text-[var(--pd-state-error)] {$$props.class}"
+          icon={faExclamationCircle} />
+      </Tooltip>
+    {:else}
+      <Tooltip top tip={error}>
+        <Fa
+          size="1.1x"
+          class="cursor-pointer text-[var(--pd-state-error)] {$$props.class}"
+          icon={faExclamationCircle} />
+      </Tooltip>
```

Comment:
it looks like we're duplicating all the code just to add tipWidth attribute.
maybe add a conditional attribute


```
in onMount:

let tipWidth = undefined;
if (actionErrorInfo) {
  tipWidth="w-max...."
}
```

and then `<Tooltip ..... tipWidth={tipWidth} ...`

Reply:
and for left and top (it should be in the onMount as well like top=false, left=false, etc)